doc/release-notes.md

```diff
@@ -78,6 +78,9 @@ New RPCs
 New settings
 ------------
 
+- RPC Whitelist system. It can give certain RPC users permissions to only some RPC calls.
```

Comment:
suggest: An RPC whitelisting feature has been added to prevent unintended access on a per-user, per-call basis to help enforce application policies for services built on Bitcoin Core.

Reply:
I think this is a bit to "user-friendly".
I think my formulation brings it more on the point as I think that the term "permission" should be included.